flow/visualize/visualizer_rllib.py

```diff
@@ -239,6 +240,13 @@ def visualizer_rllib(args):
             rets.append(ret)
         outflow = vehicles.get_outflow_rate(500)
```

Comment:
Can we add a warning here if the total horizon is less than this? If it is, these numbers might be a good deal skewed.